### DISCUSSION TITLE
Make squidfunk.github.io redirect to /mkdocs-material/

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6418] [Ideas]
My browser suggests me squidfunk but doesn't add `/mkdocs-material/` would be cool to have a personal page with links to your projects OR to your GitHub readme OR just https://squidfunk.github.io/mkdocs-material

**Comment by alexvoss** [comment]
I know that Firefox likes to complete the domain-name part and offers the full completion only somewhere down the list. I seem to remember this was mentioned before and there was a good reason to keep the current hosting arrangements. Can't find it just now but perhaps someone else will.

Just tried a few other browsers and Safari showed the same behavior as Firefox for me, while Chrome does seem to suggest the complete URL after a few visits.

**Comment by squidfunk** [comment]
Of course ... when I find some time. Maybe next life? 😅

Joking aside – I'll try to fix this in the future, but I currently have absolutely no bandwidth, sorry.

**Comment by couleurm** [comment]
yeah i'll just make a bookmark for now, thanks!
